system/mp/mpbase.hpp

```diff
@@ -134,6 +134,8 @@ extern mp_decl IGroup *createIGroup(rank_t num,INode **);
 extern mp_decl IGroup *createIGroup(rank_t num,const SocketEndpoint *); 
 extern mp_decl IGroup *createIGroup(SocketEndpointArray &);
 extern mp_decl IGroup *createIGroup(const char *endpointlist,unsigned short defport=0); // takes socketendpointlist or result of toText
+constexpr unsigned defaultGroupResolveTimeout = 10*1000*60;
```

Comment:
10 minutes ?  Does that seem like the right amount ?

Reply:
it's okay I think, it won't be normal to hit this.
The time I hit it was when spinning up a container and the group contained a hostname which was a service and K8s hadn't made it available yet.
It could take some time..